### PR TITLE
docs: Bypass `Permission denied` error on Windows when cloning the dagger repo

### PR DESCRIPTION
**Pull request by gyenabubakar**
On the Getting Started article [CI/CD in your local dev](https://docs.dagger.io/1200/local-dev), running the `git clone` command on Windows may yield a `Permission denied` error (it did for me). I appended a short note to the tip under that command telling the reader to run their terminal with elevated privileges.

image

### CODE REVIEW

**Comment by marcosnils** [comment]
:wave: thanks for opening an issue @gyenabubakar! How about if we add a link to the official `git-for-windows` documentation (https://github.com/git-for-windows/git/wiki/Symbolic-Links#allowing-non-administrators-to-create-symbolic-links) about this instead of telling the users to run a terminal with elevated privileges?

**Comment by gyenabubakar** [comment]
@marcosnils even better! I made the edit to match your suggestion.

File: pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package discord
```
Comment (TomChv) [inline pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue:1]: Those are changes from another PR, can you rebase your PR?

**Review by TomChv — COMMENTED** [review]
LGTM, just rebase your PR and we should merge it as soon as possible

File: pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue (hunk 1 above)
Comment (gyenabubakar) [reply pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue]: @TomChv I tried rebasing but the DCO check keeps failing. Facing some difficulties with Git too. I'll close and open another PR.

**Pull request closed without merging**